.circleci/config.yml

```diff
@@ -0,0 +1,111 @@
+aliases:
+  - &restore-cache
+    keys:
+      - dependencies-{{ .Branch }}-{{ checksum "package.json" }}
+      # Fallback in case checksum fails
+      - dependencies-{{ .Branch }}-
+
+  - &save-cache
+    paths:
+      - node_modules
+      - website/node_modules
+    key: dependencies-{{ .Branch }}-{{ checksum "package.json" }}
+
+  - &yarn-install
+    run: |
+      sudo npm i -g yarn@^0.27.5
```

Comment:
isn't it a super old version of yarn though?
i remember i had to manually install it on travis